packages/e2e-tests/specs/editor/blocks/navigation.test.js

```diff
@@ -435,7 +435,7 @@ describe( 'Navigation', () => {
 		expect( await getNavigationMenuRawContent() ).toMatchSnapshot();
 	} );
 
-	it( 'allows pages to be created from the navigation block and their links added to menu', async () => {
+	it.skip( 'allows pages to be created from the navigation block and their links added to menu', async () => {
```

Comment:
@talldan As seen on the latest commits on trunk, this test is failing too often, I'm skipping it for now.

Reply:
It's been failing for a while now. I suggested skipping it in [slack last week](https://wordpress.slack.com/archives/C02QB2JS7/p1640246315208600).

Reply:
I've created #37729 that skips this test as it's blocking #36746 as well.

Reply:
Apologies, I did get some failures on this when working on it, but I thought I'd solved them all as they passed multiple times locally.

Some discussion on fixing the test - https://github.com/WordPress/gutenberg/pull/37755.